### ISSUE TITLE
[Feature request] Vertical formatting for attributes

### ISSUE THREAD

**Issue by blyxyas** [rust-lang/rustfmt#5646]
Hi, it would be really cool if an attribute's content could be arranged in more ways. (Similar to `array_width`)

### Example:
Currently, if you have this attribute:

```rust
#[diagnostic(
    code(...),
    url(...),
    severity(...)
)]
fn x() {...}
```

It will get rearranged to:

```rust
#[diagnostic(code(...), url(...), severity(...))]
fn x() {...}
```

It would be useful if there was an option specifically for vertical formatting in attributes' content

**Comment by ytmimi** [comment]
Hey @blyxyas thanks for reaching out.

A lot of the work needed to implement this feature is currently being added in #5337. Once that PR lands this should be fairly straightforward to implement, but until then let's keep this on hold.

In the meantime you might try to see if you can use [`attr_fn_like_width`](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#attr_fn_like_width) width to achieve the formatting you're looking for
